eng/scripts/run_tests.ps1

```diff
@@ -0,0 +1,55 @@
+Param(
+    [string] $serviceDir
+)
+
+$cwd = Get-Location
+
+# 0. Find all test directories
+Write-Host "Finding test directories in 'sdk/$serviceDir'"
+$testDirs = & $PSScriptRoot/get_test_dirs.ps1 -serviceDir sdk/$serviceDir
+# Issues here, not returning any objects
+Write-Host "Found test directories $testDirs"
+
+$runTests = $false
+# 0b. Verify there are test files with tests to run in at least one of these directories
+$testDirs | ForEach-Object {
+    Get-ChildItem -Path $_ -Filter *_test.go | ForEach-Object {
+        if (Select-String -path $_ -pattern 'Test' -SimpleMatch) {
```

Comment:
I think `-pattern 'func Test` would be a more exact thing to match against.

Reply:
Some places use the `testify` suite capabilities and those tests don't have that pattern so for now I don't think that's the best way to do it